.changeset/famous-crabs-laugh.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-common': patch
+'@backstage/plugin-scaffolder-node': patch
```

Comment:
Let's have a separate changeset for the scaffolder, the below text doesn't really make sense in that context

Reply:
Done 🙌🏻